### DISCUSSION TITLE
Issue with adding the winget repo

### DISCUSSION THREAD

**Discussion by SLothinatorSYS** [thilojaeggi/WinGetty#36] [General]
Hi!

So I am trying out WinGetty and when trying to add the repo I an error saying "Failed to open the added source" I followed the instructions but maybe it doesn't work as it is hosted locally.
![image](https://github.com/thilojaeggi/WinGetty/assets/151180177/1906b8a1-3a45-4bfd-8a01-d66df0e6a437)

**Comment by thilojaeggi** [comment]
You need an SSL certificate.

**Reply by SLothinatorSYS** [reply]
I did create a localhost SSL certificate but doesn't seem to make any difference. anything I am missing?
![image](https://github.com/thilojaeggi/WinGetty/assets/151180177/b8de650e-3ac3-4f90-9496-936d13e580c8)

**Reply by thilojaeggi** [reply]
Does WinGetty use this certificate? Did you put it behind a. proxy that terminates SSL?
